src/modules/colorPicker/ColorPickerUI/Views/ZoomView.xaml

```diff
@@ -54,7 +54,7 @@
                 </Border.Style>
                 <Button Height="40"
                         Width="40"
-                        Content="&#xE10A;"
+                        Content="&#xe894;"
```

Comment:
Why not "ChromeClose" here? (looking at context)

Reply:
Thanks!